test/RouteMiddlewareTest.php

```diff
@@ -621,100 +621,29 @@ public function testInjectsRouteResultAsAttribute()
         $this->assertTrue($triggered);
     }
 
-    public function testMiddlewareTriggersObserversWithSuccessfulRouteResult()
+    /**
+     * @error-handling
+     */
+    public function testRoutingMiddlewareShouldReturn405ResponseDirectlyWhenRaiseThrowablesFlagEnabled()
     {
-        $matches    = ['id' => 'IDENTIFIER'];
-        $triggered  = false;
-        $middleware = function ($request, $response, $next) {
-            return $response;
-        };
-        $next = function ($request, $response, $err = null) {
-            $this->fail('Should not hit next');
-        };
-
         $request  = new ServerRequest();
         $response = new Response();
-        $result   = RouteResult::fromRouteMatch('resource', $middleware, $matches);
+        $result   = RouteResult::fromRouteFailure(['GET', 'POST']);
```

Comment:
Should we use here constants `Fig\Http\Message\RequestMethodInterface::GET` and `Fig\Http\Message\RequestMethodInterface::POST`?